docs/source/index.rst

```diff
@@ -13,6 +13,7 @@ Contents:
 
    overview
    install
+   user_guide
```

Comment:
Yep, this is def going to break ReadTheDocs.  Another evidence we should build the doc when running the tests...